examples/with-electron/README.md

```diff
@@ -1,8 +1,8 @@
 # Electron application example
 
-This example show how you can use Next.js inside an Electron application to avoid a lot of configuration, use Next.js router as view and use server-render to speed up the initial render of the application.
+This example shows how you can use Next.js inside an Electron application to avoid a lot of configuration. It uses Next.js router as view and server-render to speed up the initial render of the application.
```

Comment:
```suggestion
This example shows how you can use Next.js inside an Electron application to avoid a lot of configuration. It uses the Next.js router as view and server-render to speed up the initial render of the application.
```